.storybook/themes/news.scss

```diff
@@ -1329,6 +1329,28 @@
 						),
 					),
 				),
+				xl-promo: (
+					components: (
+						heading: (
+							font-family: var(--font-family-primary),
+							font-size: var(--global-font-size-10),
+							font-weight: var(--global-font-weight-7),
+							line-height: 1.3,
+						),
+						link: (
+							font-family: inherit,
+							font-size: inherit,
+							font-weight: inherit,
+							line-height: inherit,
+						),
+						separator: (
+							padding: 0 var(--global-spacing-2),
```

Comment:
Looks ok, but I didn't see anything in Figma to indicate this one. 🤷🏼

Reply:
I just used existing spacing taken from card-list... and it felt right.